test/testParachain.t.sol

```diff
@@ -23,26 +23,32 @@ contract ParachainTest is Test {
     address public fakeStakingContract = address(0x9999);
     address fakeStaker = address(0xabcd);
     bytes fakeReporter = abi.encode(fakeStaker); // fake reporter account on oracle consumer parachain
+    uint32 fakeWeightToFee = 5000;
+    uint8 fakeDecimals = 10;
 
     // Parachain registration
     uint32 public fakeParaId = 12;
     uint8 public fakePalletInstance = 8;
 
     StubXcmTransactorV2 private constant xcmTransactor = StubXcmTransactorV2(XCM_TRANSACTOR_V2_ADDRESS);
     StubXcmUtils private constant xcmUtils = StubXcmUtils(XCM_UTILS_ADDRESS);
+    // setting feeLocation as native token of destination chain
```

Comment:
Please remove this comment (and all similar) as it depends on the setting above as to what asset is being used.